cmd/createGitlab.go

```diff
@@ -214,6 +214,13 @@ var createGitlabCmd = &cobra.Command{
 
 			informUser("Waiting for Gitlab dns to propagate before continuing", globalFlags.SilentMode)
 			gitlab.AwaitHost("gitlab", globalFlags.DryRun)
+
+			domain := viper.GetString("aws.hostedzonename")
+			isValidCertificate, err := pkg.AwaitValidLetsEncryptCertificateNTimes(domain, globalFlags.DryRun, 60)
+			if err != nil || !isValidCertificate {
+				log.Println(err)
+			}
+
 			informUser("Pushing gitops repo to origin gitlab", globalFlags.SilentMode)
 			// refactor: sounds like a new functions, should PushGitOpsToGitLab be renamed/update signature?
```

Comment:
Let's avoid to refactor the the places we call this method. It is a new function, for selected places. Let's focus on just adding the new function on this PR. I newer PR after we stabilize main will use this function on selected places. 

After some testing of behavior impact.